utils/frame/frame-utilities-cli/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "frame-utilities-cli"
+version = "2.0.0-rc2"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+description = "CLI utils"
+documentation = "https://docs.rs/frame-utils"
+
+[dependencies]
+sp-core = { version = "2.0.0-rc2", path = "../../../primitives/core" }
+sc-cli = { version = "0.8.0-rc2", path = "../../../client/cli" }
+sp-runtime = { version = "2.0.0-rc2", path = "../../../primitives/runtime" }
+structopt = "0.3.8"
+codec = { version = "1.3.0", package = "parity-scale-codec" }
+pallet-balances = { version = "2.0.0-rc2", path = "../../../frame/balances" }
+pallet-indices = { version = "2.0.0-rc2", path = "../../../frame/indices" }
```

Comment:
I expect these two to be `optional = true` and only activated if the named features were activated.